vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/Map.java

```diff
@@ -321,4 +327,44 @@ public double getZoom() {
     public void setZoom(double zoom) {
         getView().setZoom(zoom);
     }
+
+    /**
+     * Zooms and pans the map to fit the given features in the viewport. Uses a
+     * default padding of 50 pixels and animation duration of 400 milliseconds.
+     *
+     * @param features
+     *            the features to fit in the viewport
+     */
+    public void zoomToFit(List<Feature> features) {
```

Comment:
Ideally this would also be available on the `View` instance of the map. However, none of the configuration objects have any reference to the Map element to trigger JS calls. I'm not sure about adding one at the moment, as technically configuration objects can be shared between multiple map instances (though sharing the view seems extremely unlikely).

Still, I think it's fine to start only with this for now. The new API at least fits with having `setCenter` and `setZoom` methods on the `Map`. If this turns out to be an issue, we can spend more time on making it work through the view instance.